conftest.py

```diff
@@ -29,6 +29,13 @@ def cleanup_test_files():
     shutil.rmtree(test_file_path)
 
 
+@pytest.fixture
+def set_max_bytes():
+    TestConfig.MAX_FILE_SIZE_BYTES = int(10 ** 3)
+    yield
+    TestConfig.MAX_FILE_SIZE_BYTES = int(10 ** 9)
```

Comment:
If `TestConfig.MAX_FILE_SIZE_BYTES` value is changed in the config file in future then we'd have to update it here as well. Is it worth capturing the original config in a variable here so we can reset it when returning after the test?

Reply:
I've create a preset at the top, is it what you expected?

Reply:
Nice!